modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/index.md

```diff
@@ -1,25 +1,19 @@
-# Tensor Mechanics Module class=center style=font-weight:500;font-size:200%;
+# Tensor Mechanics Module
 
-!media tensor_mechanics/lwr_3quarter_demo.png
-       id=lws_three_quarter_model
-       style=width:300px;padding-left:20px;float:right;
-       caption=Thermo-mechanical stress analysis of a reactor pressure vessel.
+- [System Documentation List](tensor_mechanics/systems.md)
 
-The +Tensor Mechanics module+ is a library of simulation tools that solve
+The Tensor Mechanics module is a library of simulation tools that solve
 continuum mechanics problems. It provides a simple approach for implementing
 even advanced mechanics models:
 
 - Plug-n-play design enables users to encorporate the relevant physics for specific and varied simulations
 - Tensor implementation matches mathematical theory
 - Straight-forward procedure for adding new physics
-- Complete [System Reference List](tensor_mechanics/systems.md) for the tensor mechanics code
+- Complete [Module Documentation List](tensor_mechanics/systems.md) for the tensor mechanics code
```

Comment:
Maybe this line isn't needed since there's a link to the same thing up above now?